SCClassLibrary/Common/Control/Bus.sc

```diff
@@ -55,6 +55,7 @@ Bus {
 	}
 
 	setMsg { arg ... values;
+		if(index.isNil) { Error("Cannot construct a % for a % that has been freed".format(thisMethod.name, this.class.name)).throw };
 		if(this.isSettable, {
```

Comment:
it would be good if you keep the same formatting convention in the whole document. So maybe if you like you could change `if(this.isSettable, { …` to `if(this.isSettable) { …`